andela/ah-infinity-stones-frontend
# #161966891 Bookmarks Listing

#### What does this PR do?
- Implement article bookmarks list

#### Description of Task to be completed?
- Add bookmarks components
- Add API fetch actions
- Hook components with redux store
- add bookmarks listing tests

#### How should this be manually tested?
1. `$ git clone -b ft-list-bookmarks-161966891 https://github.com/andela/ah-infinity-stones-frontend.git`  and `$ cd ah-infinity-stones-frontend`
2. Install the dependencies - `$ npm install`
3. Start the app by running `$ npm start` - This will run the app in the development mode.  
 To test the feature. Open http://localhost:3000/login  to view it in the browser. Provide email and password and click submit. This assumes that you have visited `http://localhost:3000/signup`, created and activated your account.
4. Visit http://localhost:3000/profile  and click `Bookmarks` button. This will list all available article bookmarks for the current user.
4. Test the app by running `$ npm test`
5. Check test coverage by running `$ npm test -- --coverage`

#### What are the relevant pivotal tracker stories?
[#161966891](https://www.pivotaltracker.com/story/show/161966891)

#### Screenshots
<img width="643" alt="screenshot 2019-01-30 at 15 07 12" src="https://user-images.githubusercontent.com/12385482/51980325-c598bf00-24a0-11e9-91a7-b1af6ee24f01.png">
